### PR TITLE
Fix idempotency key error on Android

### PR DESCRIPTION
**Pull request by SebghatYusuf**
This fix will solve the #311, I've tested many times and it works perfectly.

### CODE REVIEW

**Review by yelkamel — APPROVED** [review]

**Review by yelkamel — APPROVED** [review]

**Comment by ws6ramair** [comment]
Thanks for the fix @SebghatYusuf!

@jonasbark, can this be approved/merged? Thanks!

File: android/src/main/java/com/gettipsi/stripe/StripeModule.java [hunk 1]
```diff
@@ -201,7 +201,7 @@ public void createTokenWithCard(final ReadableMap cardData, final Promise promis
 
       mStripe.createCardToken(
         createCard(cardData),
-        mPublicKey,
+        // mPublicKey,
```
Comment (ws6ramair) [inline android/src/main/java/com/gettipsi/stripe/StripeModule.java:204]: Thanks for taking the initiative and putting out a fix for this.

Minor nit, I might recommend removing this parameter altogether (as opposed to commenting it out) and adding a comment referencing #311 for future context.

**Review by ws6ramair — APPROVED** [review]
Approved with minor comment.

**Comment by palashsethiyab3** [comment]
> This fix will solve the #311, I've tested many times and it works perfectly.


How to resolve in flutter

File: android/src/main/java/com/gettipsi/stripe/StripeModule.java (hunk 1 above)
Comment (SebghatYusuf) [reply android/src/main/java/com/gettipsi/stripe/StripeModule.java:204]: Okay sure, I will  remove the parameter.

**Comment by ws6ramair** [comment]
Hi @jonasbark, I'd be happy to donate some funds to get some expedited priority on reviewing/approving/merging this PR.

Would you be ok with the $15 onetime fix =)? 

https://github.com/sponsors/jonasbark

**Comment by jonasbark** [comment]
sure

File: android/src/main/java/com/gettipsi/stripe/StripeModule.java (hunk 1 above)
Comment (SebghatYusuf) [reply android/src/main/java/com/gettipsi/stripe/StripeModule.java]: it's resolved.

**Comment by ws6ramair** [comment]
Thanks @jonasbark! Just sent the onetime sponsorship money. Thanks!

Let me know if you have any feedback on the approach that @SebghatYusuf so kindly implemented as well.

**Comment by ws6ramair** [comment]
Thanks so much @SebghatYusuf and @jonasbark for helping to get this in. Just tested out `1.1.4` and confirmed everything works as expected.

**Pull request merged**